x-pack/tsconfig.json

```diff
@@ -37,12 +37,7 @@
       "plugins/*": ["src/legacy/core_plugins/*/public/"],
       "fixtures/*": ["src/fixtures/*"],
     },
-    "types": [
-      "node",
-      "jest",
-      "flot",
-      "jest-styled-components",
-      "@testing-library/jest-dom"
-    ]
+    // overhead is too significant
+    "incremental": false,
```

Comment:
`incremental: true` first pass
```
real	12m22.044s
user	12m47.693s
sys	0m20.303s
```
`incremental: true` second pass
```
real	1m16.779s
user	1m14.341s
sys	0m7.595s
```

`incremental: true` after changing a file
```
real	5m11.158s
user	4m46.419s
sys	0m12.713s
```

`incremental: false`
```
real	6m30.505s
user	6m34.917s
sys	0m14.650s
```

Reply:
So we'll need to break up x-pack into multiple projects to get benefits here, correct?

Reply:
@joshdover yes, as per https://github.com/elastic/kibana/issues/46773#issuecomment-678373634